tests/autoyast/installation.pm

```diff
@@ -118,17 +118,18 @@ sub run {
             $num_errors++;
         }
         elsif (match_has_tag('warning-pop-up')) {
-            # Softfail only on sle, as timeout is there on CaaSP
+            die "Unknown popup message" unless check_screen('autoyast-known-warning', 0);
+
+            # Wait until popup disappears
+            my $disappered = wait_screen_change { sleep 11 };
+            # Softfail only on sle, as timeout is there on CaaSP. Verify that screen doesn't change, no timeout
             if (check_var('DISTRI', 'sle') && check_screen('warning-partition-reduced', 0)) {
                 # See poo#19978, no timeout on partition warning, hence need to click OK button to soft-fail
                 record_soft_failure('bsc#1045470');
```

Comment:
Maybe use record_info instead of record_soft_fail (I believe nobody will fix this bug, I will close it).